packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,37 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
+    description:
+      'Storybook is a frontend workshop for building UI components and pages in isolation.',
+    website: 'https://storybook.js.org',
+    ignoreRuntimes: ['@vercel/next', '@vercel/node'],
+    detectors: {
+      every: [
+        {
+          matchPackage: 'storybook',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install`, `pnpm install`, or `npm install`',
+      },
+      buildCommand: {
+        value: 'storybook build',
+      },
+      devCommand: {
+        value: `start-storybook -p $PORT`,
+      },
+      outputDirectory: {
+        value: 'storybook-static',
```

Comment:
Is the output dir configurable? We might need to read the config file to handle all cases

Reply:
It doesn't look like Storybook has a config file. The `storybook build` command supports a `--output-dir [path]` option. The default value is `"storybook-static"`: https://github.com/storybookjs/storybook/blob/e7817e82757618992f930f6157ddb68fc7541260/code/lib/cli/src/build.ts#L9.